_pages/docs/next-ls/editors.md

```diff
@@ -14,11 +14,17 @@ github_stars: elixir-tools/next-ls
 
 ## Neovim
 
-[elixir-tools.nvim](https://github.com/elixir-tools/elixir-tools.nvim) is a first party elixir-tools editor plugin and will install and manage Next LS for you
+- [elixir-tools.nvim](https://github.com/elixir-tools/elixir-tools.nvim) is a first party elixir-tools editor plugin and will install and manage Next LS for you
+- You might also decide to install and manage `next-ls` yourself using `nvim-lspconfig`, and perhaps, [`homebrew`](/docs/next-ls/installation/#homebrew) for the `nextls` binary.
+  - ```lua
+    require("lspconfig")["nextls"].setup({
+      cmd = {"nextls", "--stdio"} -- assumes you have `nextls` in your `$PATH`
+    })
+    ```
 
```

Comment:
instead of a bulleted list, lets use h3 sections for elixir-tools.nvim and nvim-lspconfig, similar to how the emacs sections are.

this also allows you to deep link to each section

Reply:
Also, if you could add the example of how to configure the initialization options, that would be great too